src/website/app/demos/Avatar/examples/colors.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import Avatar from '../../../../../Avatar';
+import DemoLayout from '../components/DemoLayout';
+
+export default {
+  id: 'colors',
+  title: 'Colors',
+  description: `Avatar is available in any of the
+[palette's base colors](/color/#guidelines-ramps). The \`"theme"\` value, which
+is default, will automatically use your theme's base color.`,
+  scope: { Avatar, DemoLayout },
+  source: `
+    <DemoLayout>
+      <Avatar color="theme"  >Theme  </Avatar>
```

Comment:
Again to consider not having a "theme" color name, but rather having it default to the theme color.